app/views/components/_markdown-editor.html.erb

```diff
@@ -0,0 +1,52 @@
+<%
+  label_text ||= "Enter your markdown here"
+  textarea ||= {}
+  error_items ||= []
+   component_classes = %w[app-c-markdown-editor govuk-form-group]
```

Comment:
Nitpick: I don't think this should be indented.

Reply:
Uodated